python/python-hello-world/src/Dockerfile

```diff
@@ -14,4 +14,4 @@ RUN pip install --trusted-host pypi.python.org -r requirements.txt
 COPY . .
 
 # Run app.py when the container launches
-ENTRYPOINT ["python", "-m", "ptvsd", "--port", "3000", "--host", "0.0.0.0", "app.py"]
```

Comment:
Oh nice these aren't required anymore? I think we can also clean up the `Procfile`s as well